### DISCUSSION TITLE
AWS Aurora (PostgreSQL) SQL ETL Cannot establish a connection

### DISCUSSION THREAD

**Discussion by nickchampion** [ravendb/ravendb#15612] [General]
Hi,

I am trying to set up an SQL ETL task but cannot connect to my PostgreSQL database, I am using AWS Aurora with a PostgreSQL compatible database and am able to connect to the database from a node.js app but the connection fails when I try to create a connection string in the studio.

The connection string format I am using is

Host=cluster.eu-west-2.rds.amazonaws.com;Database=mydatabase;Username=user1;Password=password;Port=5432

Looking through the server side code and NgpSql code this looks like it should work, I've tried quoting the host.

Am I missing something, does SQL ETL support Aurora?

Thanks

**Comment by arekpalinski** [comment]
What error do you get when you try to `Test Connection`?

**Comment by nickchampion** [comment]
Hi @arekpalinski a timeout error

<img width="1031" alt="image" src="https://user-images.githubusercontent.com/3949005/212009354-9e44d492-5e62-4204-bbe0-8f881eade698.png">

**Reply by arekpalinski** [reply]
Any chance it's a firewall issue? Can you check if  port `5432` is open using `telnet` from the machine where RavenDB is running?

**Reply by nickchampion** [reply]
I'm running RavenDB in the cloud, the Aurora cluster is public and I am able to connect from my local machine, I can share the connection string with you privately if it will help there is nothing in the database at the moment, can I DM you it somehow?

**Reply by nickchampion** [reply]
Is there something I need to enable on my cloud instances to enable this?

**Reply by arekpalinski** [reply]
Please open support ticket via RavenDB Cloud portal and we'll check that

**Comment by arekpalinski** [comment]
The problem was the configuration of the firewall on AWS.
